ui/pages/import-tokens/index.scss

```diff
@@ -0,0 +1,85 @@
+
+.scrollable {
+  ::-webkit-scrollbar {
+    display: none; /* for Chrome, Safari, and Opera */
+  }
+}
+
+.banner {
+  margin-left: 20px;
+  margin-right: 20px;
+}
+
+
+.import-tokens-page {
+  padding: 16px;
```

Comment:
Can this be `marginLeft ` and `marginRight` on a `<Box>` component somewhere?

Reply:
Removed not needed 🙏